api/net/Areas/Services/Controllers/ContentController.cs

```diff
@@ -419,8 +419,12 @@ public async Task<IActionResult> AttachQuotes([FromRoute] long id, [FromBody] IE
         if (!quotes.Any()) return new JsonResult(new ContentModel(content, _serializerOptions));
 
         _quoteService.Attach(Array.ConvertAll(quotes.ToArray(), item => (Quote)item));
-        
+
         content = _contentService.FindById(id);
+        if (content == null) return new JsonResult(new { Error = "Content does not exist" })
```

Comment:
Please handle nullable objects.  The linter always tells you about these if you use it.